copilot/fastapi/run_tests.sh

```
@@ -0,0 +1,85 @@
+echo -e "${YELLOW}Installing/upgrading required packages...${NC}"
+
+# First uninstall any existing pytest and pytest-asyncio to avoid version conflicts
+python -m pip uninstall -y pytest pytest-asyncio pytest-cov
+
+# Install specific compatible versions
+python -m pip install pytest==7.3.1 pytest-asyncio==0.21.1 pytest-cov==4.1.0 httpx pyyaml kubernetes fastapi uvicorn -q
+
+# Debug: Print the file content of conftest.py to understand the import issue
+if [ -f "tests/conftest.py" ]; then
+    echo -e "${YELLOW}Content of tests/conftest.py:${NC}"
+    cat tests/conftest.py | head -n 20
```

Shellcheck (0.10.0)

[style] Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.

(SC2002)